client/app/src/main/res/layout/activity_main.xml

```diff
@@ -2,21 +2,42 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/tempButton"

```

Comment:
This too. Revert the changes from mainActivity

Reply:
yea, most of it came from the profileActivity. I built on in. MainActivity is essentially currently our navigation page. MainActivity doesnt always have to be the Start activity though.